docs/examples/postgres.md

```diff
@@ -0,0 +1,13 @@
+# Reusable Postgres Database
+
+## Introduction
+
+This example shows the usage of the postgres module's Snapshot feature to give each test a clean database without having
```

Comment:
I have this idea: we can add an `Examples` section for each module, as the last element in the menu for the postgres module. Then I'd move the entire content of this file there, looking like this:

```md
## Examples

### Using Snapshots
This example shows the usage of the postgres module's Snapshot feature to give each test a clean database without having
to recreate the database container on every test or run heavy scripts to clean your database. This makes the individual
tests very modular, since they always run on a brand-new database.

<!--codeinclude-->
[Test with a reusable Postgres container](../../modules/postgres/postgres_test.go) inside_block:snapshotAndReset
<!--/codeinclude-->
```

Wdyt?

Reply:
The thing in the Go project is that we have pkg.go.dev, so all the testable examples land there. That's why I like testable examples, although a balance between real tests and examples is needed.